app/controllers/api/v2/compliance/oval_reports_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Api
+  module V2
+    module Compliance
+      class OvalReportsController < ::Api::V2::BaseController
+        include Foreman::Controller::SmartProxyAuth
+        add_smart_proxy_filters :create, :features => 'Openscap'
+
+        skip_before_action :setup_has_many_params
+        before_action :find_resources_before_create, :only => [:create]
+
+        api :POST, "/compliance/oval_reports/:cname/:oval_policy_id/:date", N_("Upload an OVAL report - a list of CVEs for given host")
+        param :cname, :identifier, :required => true
+        param :oval_policy_id, :identifier, :required => true
+        param :date, :identifier, :required => true
+
+        def create
+          ForemanOpenscap::Oval::Cves.new.create(@host, params.to_unsafe_h)
+          if @host.errors.any?
+            upload_fail @host.errors.full_messages
+          else
+            @host.refresh_statuses([HostStatus.find_status_by_humanized_name("oval")])
```

Comment:
`@host.refresh_statuses([HostStatus.find_status_by_humanized_name("oval")])` this no longer works, it should be
`@host.refresh_statuses([ForemanOpenscap::OvalStatus])`